src/snyk/common/languageServer/middleware.ts

```diff
@@ -45,7 +55,42 @@ export class LanguageClientMiddleware implements Middleware {
       return [serverSettings];
     },
   };
+  window: WindowMiddleware = {
+    showDocument: async (params: ShowDocumentParams, next) => {
+      let uri;
+      try {
+        uri = new URL(params.uri);
+      } catch (error) {
+        throw new Error('Invalid URI recieved for window/showDocument');
```

Comment:
Typo ;). Maybe this should just silently fail instead of throwing an error, as it's a decorating middleware we are adding.